.rubocop.yml

```diff
@@ -44,10 +44,14 @@ Layout/SpaceInsideBrackets:
   Enabled: false
 Lint/EndAlignment:
   Severity: error
+Lint/RescueWithoutErrorClass:
+  Enabled: false
```

Comment:
Oooo we should enable this.

Reply:
Yeah, but it's always better to rescue explicitly what you want to rescue instead of everything. We can do this in a subsequent pass.